Framework/Core/src/Base64.h

```diff
@@ -0,0 +1,19 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_FRAMEWORK_BASE64_H_
+#define O2_FRAMEWORK_BASE64_H_
+
+int base64_encode(char* dest, int size, unsigned char* src, int slen);
```

Comment:
should this be in the top level namespace?

Reply:
does it matter? it's not a public API.

Reply:
anyways, I moved it in a namespace.